Makefile.am

```diff
@@ -624,6 +624,10 @@ NETDATA_COMMON_LIBS = \
     $(OPTIONAL_EBPF_LIBS) \
     $(NULL)
 
+if LINK_STATIC_JSONC
```

Comment:
This is technically correct but seems fragile (i.e. we should only end up with LINK_STATIC_JSONC set if OPTIONAL_JSONC_LIBS is empty). What about the following to make the relationship guaranteed?
```
If LINK_STATIC_JSONC
     NETDATA_COMMON_LIBS += externaldeps/jsonc/libjson-c.a
else
    NETDATA_COMMON_LIBS += $(OPTIONAL_JSONC_LIBS)
endif
```